rebasehelper/tests/functional/conftest.py

```diff
@@ -57,4 +59,4 @@ def make_logs_report():
 def pytest_runtest_makereport():
     outcome = yield
     report = outcome.get_result()
-    report.sections.append(('Logs', make_logs_report()))
+    report.sections.append(('Logs', make_files_report()))
```

Comment:
The section should be also renamed. Actually, now that I think about it, various files created by tests are usually referred to as artifacts. Perhaps that would be the best term to use here, instead of files or logs.